jest.config.ts

```diff
@@ -5,11 +5,6 @@ const jestConfig: JestConfigWithTsJest = {
   moduleNameMapper: {
     "^(\\.{1,2}/.*)\\.js$": "$1",
   },
-  testMatch: ["**/test/**/*.test.ts", "**/test/tests.ts"],
-  coverageProvider: "v8",
-  coveragePathIgnorePatterns: ["/node_modules/", "./test"],
-  reporters: ["default", ["jest-junit", { outputDirectory: "test-results/jest" }]],
-  coverageReporters: ["lcov", "text", "text-summary"],
```

Comment:
Let's bring back the coverageReporters, as this changes the coverage output in the terminal.

Reply:
Done 👍